application/helpers/expressions/em_manager_helper.php

```
@@ -6682,10 +6682,18 @@ function ($sqRankAnwsers) {
             }
         }
 
-        // Process Default : 1st part : update in DB if actually relevant and not already set
+        // Process Default and prefilled values : 1st part : update in DB if actually relevant and not already set
         if ($qrel && $grel) {
             $allSQs = explode('|', (string) $LEM->qid2code[$qid]);
             foreach ($allSQs as $sgqa) {
+                /* prefilled by URL but deleted by relevance */
+                if (!isset($_SESSION[$LEM->sessid][$sgqa]) && isset($_SESSION[$LEM->sessid]['startingValues'][$sgqa])) {
+                    $startingValue = $_SESSION[$LEM->sessid]['startingValues'][$sgqa];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$startingValue'. (undefined)

(UnusedLocalVariable)